1.9/api/pkgpanda.yaml

```diff
@@ -0,0 +1,207 @@
+swagger: '2.0'
+info:
+  title: DC/OS Pkgpanda API
+  version: '1.0.0'
+basePath: /pkgpanda
+
```

Comment:
Looks great, nice to have this!! Need to add "tags" to ensure the proper display and grouping. I think it would be something like the following:
```
tags:
  - name: repository
    description: Pithy description
  - name: active
    description: Pithy description
```
  
And then you add the tags to the endpoint aka resource definition under `description` like this:
```
     tags:
        - repository
```

Let me know if you need more information. You can see some examples in the Marathon YAML here https://github.com/dcos/dcos-docs/blob/master/1.9/api/marathon.yaml Although I am seeing weird `^M` characters after each line, not sure what that's about.